script.js

```diff
@@ -77,3 +77,107 @@ function myFunction(x) {
+  document.getElementById('postTitle').innerText = showItemInfo.title;
+  document.getElementById('tech1').innerHTML = showItemInfo.technologies.tech1;
+  document.getElementById('tech2').innerHTML = showItemInfo.technologies.tech2;
+  document.getElementById('tech3').innerHTML = showItemInfo.technologies.tech3;
+  document.getElementById('description').innerText = showItemInfo.description;
+  document.getElementById('image').style.backgroundImage = `url('${showItemInfo.image}')`;
+  document.getElementById('liveButton').setAttribute('href', showItemInfo.liveVersion);
+  document.getElementById('liveSource').setAttribute('href', showItemInfo.liveSource);
+});
+showModal4.addEventListener('click', () => {
+  document.getElementById('modal').classList.add('active');
+  document.getElementById('postTitle').innerText = showItemInfo.title;
+  document.getElementById('tech1').innerHTML = showItemInfo.technologies.tech1;
+  document.getElementById('tech2').innerHTML = showItemInfo.technologies.tech2;
+  document.getElementById('tech3').innerHTML = showItemInfo.technologies.tech3;
+  document.getElementById('description').innerText = showItemInfo.description;
+  document.getElementById('image').style.backgroundImage = `url('${showItemInfo.image}')`;
+  document.getElementById('liveButton').setAttribute('href', showItemInfo.liveVersion);
+  document.getElementById('liveSource').setAttribute('href', showItemInfo.liveSource);
+});
+showModal5.addEventListener('click', () => {
+  document.getElementById('modal').classList.add('active');
+  document.getElementById('postTitle').innerText = showItemInfo.title;
+  document.getElementById('tech1').innerHTML = showItemInfo.technologies.tech1;
+  document.getElementById('tech2').innerHTML = showItemInfo.technologies.tech2;
+  document.getElementById('tech3').innerHTML = showItemInfo.technologies.tech3;
+  document.getElementById('description').innerText = showItemInfo.description;
+  document.getElementById('image').style.backgroundImage = `url('${showItemInfo.image}')`;
+  document.getElementById('liveButton').setAttribute('href', showItemInfo.liveVersion);
+  document.getElementById('liveSource').setAttribute('href', showItemInfo.liveSource);
+});
+showModal6.addEventListener('click', () => {
+  document.getElementById('modal').classList.add('active');
+  document.getElementById('postTitle').innerText = showItemInfo.title;
+  document.getElementById('tech1').innerHTML = showItemInfo.technologies.tech1;
+  document.getElementById('tech2').innerHTML = showItemInfo.technologies.tech2;
+  document.getElementById('tech3').innerHTML = showItemInfo.technologies.tech3;
+  document.getElementById('description').innerText = showItemInfo.description;
+  document.getElementById('image').style.backgroundImage = `url('${showItemInfo.image}')`;
+  document.getElementById('liveButton').setAttribute('href', showItemInfo.liveVersion);
+  document.getElementById('liveSource').setAttribute('href', showItemInfo.liveSource);
```

Comment:
- [ ] On desktop version I can't see the `See project ` button to open the pop-up please try to make the button appear  when hover on the card
![image](https://user-images.githubusercontent.com/61790587/179960375-5c28f852-ab01-47e1-83c4-0b871d26b6d5.png)